### DISCUSSION TITLE
Stored Switch Decoder Addresses

### DISCUSSION THREAD

**Discussion by LorcaSnep** [DCC-EX/EX-T3-WiFi#100] [Ideas]
I have been utilizing one of these throttles for about 6 months and it has been excellent.  I have several DS64 decoders on my layout and it would be great to be able to add stationary addresses to a roster like locomotives are.  That way you can just swap to an accessory page and quickly change a switch value.  I went away from an ESU Cab Control Command Station for multiple reasons but the stored accessory addresses is one feature I do miss about it.

**Comment by alex-code** [comment]
Sounds like a good feature.
Would it be like the current loco list but just allows you to name accessories?

Could see if I can squeeze in a group option so you could group accessories, e.g. points/turnouts

**Reply by LorcaSnep** [reply]
Yes!  Basically identical except for 2 features that I think might be a good idea.  The first being able to trigger the turnout from the list without opening up the decoder like on the locomotives.  And the second that in the configuration there is a way to invert the function so that the "normal" position of the switch can be reversed depending on the layout.
